src/modules/get-weather-data.js

```diff
@@ -0,0 +1,23 @@
+import populateData from './populate-data';
+import getBackground from './get-background';
+import invalidCity from './invalid-input';
+
+const getWeatherData = async (cityName) => {
+  try {
+    const apiKey = process.env.API_KEY;
+    const response = await fetch(`https://api.openweathermap.org/data/2.5/weather?q=${cityName}&appid=${apiKey}`);
+    const weatherData = await response.json();
+
+    if (weatherData.cod === '404') {
+      invalidCity();
+    } else {
+      localStorage.setItem('weatherData', JSON.stringify(weatherData));
+      populateData();
+      getBackground(weatherData.weather[0].main);
+    }
+  } catch {
+    // error
```

Comment:
* [ ] Let return a message instead of just writing a comment, this will help you in debugging. You could return a customized status code ie,
`let response = {};`
`response={status: 404};`